docs/Config.md

```diff
@@ -75,6 +75,7 @@ git:
   paging:
     colorArg: always
     useConfig: false
+    pager: delta
```

Comment:
This section describes lazygit's defaults (i.e. the values for the options that are used when you don't specify them). "delta" is not the default for the pager option, so it shouldn't be here.

How to configure a pager is described in docs/Custom_Pagers.md, so I don't think there's anything missing.

Reply:
I'll change it. This is the first place I looked and when I didn't find it I got confused so I think it will help someone else